config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/BuilderConstants.java

```diff
@@ -29,6 +30,13 @@ public boolean test(Entity entity) {
         }
     };
 
+    public static final Predicate<Entity> FILTER_METADATA_NON_ENV_ENTITIES = new Predicate<Entity>() {
```

Comment:
You should change name to something like `FILTER_NON_ENV_ENTITIES_EXCLUDING_FOLDER`